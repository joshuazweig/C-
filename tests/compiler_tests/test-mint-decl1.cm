int main()
{
  stone x;
  stone y;
  stone p;

  mint m;
  mint n;

  x = "15352395";
  y = "11";
  p = "65537"; 

  m = <x, p>;
  n = <y, p>;

  print_stone(x);
  print_stone(y);
  print_stone(p);

  print_mint(m);
  print_mint(n);

  return 0;
}
