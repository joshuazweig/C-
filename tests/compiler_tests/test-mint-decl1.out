15352395
11
65537
16737
65537
11
65537
